plugins/generator-1.18.1/forge-1.18.1/procedures/world_entity_inrange_foreach.java.ftl

```diff
@@ -0,0 +1,9 @@
+{
+	final Vec3 _center = new Vec3(${input$x}, ${input$y}, ${input$z});
+	List<Entity> _entfound = world.getEntitiesOfClass(Entity.class,
+		AABB.ofSize(_center, ${input$range}, ${input$range}, ${input$range}), e -> true).stream()

```

Comment:
This will not do the same as inflate, adding like this is not from the center. Keep it as it was

Reply:
This is used in other _Entity in range_ procedure blocks, why should this be different?

Reply:
Other procedures use a slightly different approach than what you use here. Stick to the approach used so far